### ISSUE TITLE
Command failed with exit code 1: ffmpeg.exe -i

### ISSUE THREAD

**Issue by mamift** [lukasbach/pensieve#3]
**Describe the bug**
Post-processing failed; Command failed with exit code 1: C:\Users\mmiftah\AppData\Local\pensieve\app-1.0.8\resources\extra\ffmpeg.exe -i 

**To Reproduce**
Import an audio file and wait for post-processing to finish

**Expected behavior**
No errors should occur

**Screenshots**
![image](https://github.com/lukasbach/pensieve/assets/2508444/29167824-2458-4216-aeb4-cb5514311272)

**Additional context**
```

Command failed with exit code 1: C:\Users\mmiftah\AppData\Local\pensieve\app-1.0.8\resources\extra\ffmpeg.exe -i C:\Users\mmiftah\AppData\Roaming\Pensieve\recordings\C:\Users\mmiftah\AppData\Roaming\Pensieve\recordings\2024-5-8_13-19-4\whisper-input.wav -f null -
ffmpeg version 3.0.1 Copyright (c) 2000-2016 the FFmpeg developers
  built with gcc 5.3.0 (GCC)
  configuration: --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libdcadec --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzimg --enable-lzma --enable-decklink --enable-zlib
  libavutil      55. 17.103 / 55. 17.103
  libavcodec     57. 24.102 / 57. 24.102
  libavformat    57. 25.100 / 57. 25.100
  libavdevice    57.  0.101 / 57.  0.101
  libavfilter     6. 31.100 /  6. 31.100
  libswscale      4.  0.100 /  4.  0.100
  libswresample   2.  0.101 /  2.  0.101
  libpostproc    54.  0.100 / 54.  0.100
C:\Users\mmiftah\AppData\Roaming\Pensieve\recordings\C:\Users\mmiftah\AppData\Roaming\Pensieve\recordings\2024-5-8_13-19-4\whisper-input.wav: Invalid argument
```

- Windows 10 21H2
- Version 1.0.8

**Comment by jpapagia** [comment]
Same issue here:

Postprocessing is finished.
Untitled recording
Command failed with exit code 1: C:\Users\JohnPapagiannopoulos\AppData\Local\pensieve\app-1.0.7\resources\extra\ffmpeg.exe -i C:\Users\JohnPapagiannopoulos\Documents\Meetings\C:\Users\JohnPapagiannopoulos\Documents\Meetings\2024-5-13_16-34-13\whisper-input.wav -f null -
ffmpeg version 3.0.1 Copyright (c) 2000-2016 the FFmpeg developers
  built with gcc 5.3.0 (GCC)
  configuration: --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libdcadec --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzimg --enable-lzma --enable-decklink --enable-zlib
  libavutil      55. 17.103 / 55. 17.103
  libavcodec     57. 24.102 / 57. 24.102
  libavformat    57. 25.100 / 57. 25.100
  libavdevice    57.  0.101 / 57.  0.101
  libavfilter     6. 31.100 /  6. 31.100
  libswscale      4.  0.100 /  4.  0.100
  libswresample   2.  0.101 /  2.  0.101
  libpostproc    54.  0.100 / 54.  0.100
C:\Users\JohnPapagiannopoulos\Documents\Meetings\C:\Users\JohnPapagiannopoulos\Documents\Meetings\2024-5-13_16-34-13\whisper-input.wav: Invalid argument

Here are my settings:
[settings.json](https://github.com/user-attachments/files/15828499/settings.json)

Windows 11
1.0.8

**Comment by lukasbach** [comment]
Thanks to both of you for the report and the details, that was very helpful! I believe to have found the issue and fixed it as of 1.0.9. The app should auto-update during the next start.
